cmd/scaler/Makefile

```diff
@@ -0,0 +1,23 @@
+PROMBENCH_CMD        = ./scaler
+DOCKER_TAG = docker.io/prombench/scaler:2.0.0
+
+scale:
+	$(PROMBENCH_CMD) scale -v PR_NUMBER:${PR_NUMBER} ${SCALE_DOWN_REPLICAS} ${SCALE_UP_REPLICAS} ${INTERVAL} \
+		-f components/prombench/manifests/benchmark/2_fake-webserver.yaml
+
+build:
+	@go version | grep go1.11 || exit  "Requires golang 1.11 with support for modules!"
+	@GO111MODULE=on GOARCH=amd64 CGO_ENABLED=0 GOOS=linux go build scaler.go
```

Comment:
how about using a small golang 1.11 image from  https://hub.docker.com/_/golang/ to build the binary? 
We wouldn't need to check for modules support and devs wouldn't need to have golang installed at all.

Reply:
I will fix this in #145